src/geometry/Bounds.js

```diff
@@ -154,9 +169,40 @@ Bounds.prototype = {
 		return xOverlaps && yOverlaps;
 	},
 
+	// @method isValid(): Boolean
+	// Returns `true` if the bounds are properly initialized.
 	isValid: function () {
 		return !!(this.min && this.max);
-	}
+	},
+
+
+	// @method pad(bufferRatio: Number): Bounds
+	// Returns bounds created by extending or retracting the current bounds by a given ratio in each direction.
+	// For example, a ratio of 0.5 extends the bounds by 50% in each direction.
+	// Negative values will retract the bounds.
+	pad: function (bufferRatio) {
+		var min = this.min,
+		max = this.max,
+		heightBuffer = Math.abs(min.x - max.x) * bufferRatio,
+		widthBuffer = Math.abs(min.y - max.y) * bufferRatio;
+
+
+		return toBounds(
+			toPoint(min.x - heightBuffer, min.y - widthBuffer),
+			toPoint(max.x + heightBuffer, max.y + widthBuffer));
+	},
+
+
+	// @method equals(otherBounds: Bounds, maxMargin?: Number): Boolean
+	// Returns `true` if the rectangle is equivalent (within a small margin of error) to the given bounds. The margin of error can be overridden by setting `maxMargin` to a small number.
+	equals: function (bounds) {
+		if (!bounds) { return false; }
+
+		bounds = toBounds(bounds);
+
+		return this.min.equals(bounds.getTopLeft()) &&
+			this.max.equals(bounds.getBottomRight());
+	},

```

Comment:
`maxMargin` is left out, contrary to the comment and the implementation in `LatLngBounds`

Reply:
Ups, The docs will be changed